website_v2/gcloud-powershell-website-v2/app.yaml

```diff
@@ -0,0 +1,18 @@
+application: gcloud-powershell-website-v2
```

Comment:
Yes. We definitely want to call out that this GAE app isn't for actually serving things. (Rather, we will use GitHub pages.) It is just to easily set up something for testing/debugging purposes.

Although... sorry this didn't dawn on me earlier, but I imagine we could just use `jekyll` (e.g. `jekyll serve . --watch`) for the same purposes. Though no point in changing it now.